Web/starter_project_vanguard/components/blog/BlogCard.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react'
+import { Blog } from '../../types/blogs'
+import Image from 'next/image'
+import { TbClockFilled } from 'react-icons/tb'
+import { FiMessageSquare } from 'react-icons/fi'
+
+interface Props {
+  blog: Blog
+}
+
+const BlogCard: React.FC<Props> = ({ blog }) => {
+  const [readMore, setReadMore] = useState<boolean>(false)
+  const numberFormater = (num : number) => {
+    if (num >= 1e6) return +(num / 1e6).toFixed(1) + "m";
+    if (num >= 1e3) return +(num / 1e3).toFixed(1) + "k";
+    if (num < 1e3) return num;
+  };
+  
+  return (
+
+    <div className="max-w-sm rounded-sm shadow">
+      <Image
+        className="rounded object-cover h-52 w-full"
+        width={400}
+        height={400}
+        src={blog.img}
+        alt={blog.title}
+      />
+
+      <div className="p-4">
+        <h5 className="mb-4 text-primary-text text-sm font-semibold">
+          {blog.title}
+        </h5>
+        <div className="flex space-x-2 text-gray-400 items-center text-[0.6rem] mb-4">
+          <Image
+            className="rounded-full object-cover h-5 w-5 items-center"
+            width={10}
+            height={10}
+            src={blog.img}
+            alt={blog.title}
+          />
+          <p>by</p>
+          <p className="font-semibold text-primary-text">{blog.authorName} </p>
+          <p>
+            <span className="pr-2">|</span>
+            {blog.date}
+          </p>
+        </div>
+
+        <div className="flex flex-row space-x-2 my-3">
+          {blog.skills.map((skill, index) => {
+            return (
+              <div
+                key={index}
+                className="rounded-full bg-neutral-100 uppercase text-[0.6rem] font-semibold text-secondary-text py-1 px-3"
```

Comment:
for the font size let's use the already existing tailwind classes

Reply:
The smallest text size in tailwind is 'text-xs', which is 0.75 rem, but the font-size mentioned on the UI is less than text-xs. That is why I used it implicitly.